src/pages/Flow/Run.vue

```diff
@@ -444,6 +465,54 @@ export default {
           </v-col>
         </v-row>
 
+        <v-row v-if="showAdvanced" class="my-2 py-8 row-divider" no-gutters>
+          <v-row class="my-2 py-8" no-gutters>
+            <v-col cols="12" md="3">
+              <div
+                class="py-0"
+                :class="{ 'pr-24': $vuetify.breakpoint.mdAndUp }"
+              >
+                <div class="text-h5">
+                  Logging Level
+                  <MenuTooltip>
+                    <p>
+                      Logging level for the flow run. If none is specified,
+                      default agent logging level will be used.
+                    </p>
+                    <p>
+                      This can also be controlled by providing an environment
+                      variable, "PREFECT__LOGGING__LEVEL".
+                    </p>
+                    <p>
+                      Any dropdown selection besides "Default" will override the
+                      environment variable.
+                    </p>
+                    <p>
+                      Please note this is only guaranteed to work for Agents
+                      running Prefect Core TODO or later.
+                    </p>
+                  </MenuTooltip>
+                </div>
+              </div>
+            </v-col>
+
+            <v-col cols="12" md="9" class="mt-n4 mt-md-0">
+              <v-select v-model="loggingLevel" outlined :items="loggingLevels">
+                <template #item="{ item }">
+                  <div>
+                    <v-chip
+                      class="ma-2 debuglevel"
```

Comment:
```suggestion
                      class="ma-2 debuglevel cursor-pointer"
```
Right now the chip element has `cursor: default` on it, which makes it seem like a different interaction, this will fix that.

Reply:
resolved in latest commit